report/renderer/components.js

```diff
@@ -567,6 +567,17 @@ function createSnippetLineComponent(dom) {
   return el0;
 }
 
+/**
+ * @param {DOM} dom
+ */
+function createStylesComponent(dom) {
```

Comment:
may need something more clever than https://github.com/GoogleChrome/lighthouse/blob/c53a030dab512c7dbafcd8a256b33ac541286a50/report/renderer/dom.js#L102 to prevent injecting the same styles when there are multiple reports on the page.

perhaps each `style` element could be given a UUID in `build-report-components.js` (`<style data-uuid=86fca206-0a7c-46b2-9dd9-c96d2b40b3c9> ... </style>`), and `DOM.createComponent` could  remember check all `data-uuid` in the fragment it's returning and remove any already on the page

Reply:
multiple injections of these global styles doesnt cause any problems.. so i think we should just roll with it for now .